docs/install-docker.md

```diff
@@ -96,17 +96,16 @@ sudo usermod -aG docker $(whoami)
 
 ```bash
 # 1. initialize repo and submodules
-git clone -b stable https://github.com/CanDIG/CanDIGv2.git
+git clone -b develop https://github.com/CanDIG/CanDIGv2.git
 git submodule update --init --recursive
+cd CanDIGv2
```

Comment:
Do we want `develop` branch for installation?

Also, I think 101 should be before 100.  I believe we have to `cd` into the cloned repo and then `submodule init`.